superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -548,68 +548,75 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     });
   };
 
-  const renderAvailableSelector = () => (
-    <div className="available">
-      <h4 className="available-label">
-        Or choose from a list of other databases we support:
-      </h4>
-      <div className="control-label">Supported databases</div>
-      <Select
-        className="available-select"
-        onChange={setDatabaseModel}
-        placeholder="Choose a database..."
-      >
-        {availableDbs?.databases
-          ?.sort((a: DatabaseForm, b: DatabaseForm) =>
-            a.name.localeCompare(b.name),
-          )
-          .map((database: DatabaseForm) => (
-            <Select.Option value={database.engine} key={database.engine}>
-              {database.name}
-            </Select.Option>
-          ))}
-      </Select>
-      <Alert
-        showIcon
-        closable={false}
-        css={(theme: SupersetTheme) => antDAlertStyles(theme)}
-        type="info"
-        message={
-          connectionAlert?.ADD_DATABASE?.message ||
-          t('Want to add a new database?')
-        }
-        description={
-          connectionAlert?.ADD_DATABASE ? (
-            <>
-              Any databases that allow connections via SQL Alchemy URIs can be
-              added.{' '}
-              <a
-                href={connectionAlert?.ADD_DATABASE.contact_link}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {connectionAlert?.ADD_DATABASE.contact_description_link}
-              </a>{' '}
-              {connectionAlert?.ADD_DATABASE.description}
-            </>
-          ) : (
-            <>
-              Any databases that allow connections via SQL Alchemy URIs can be
-              added. Learn about how to connect a database driver{' '}
-              <a
-                href={DOCUMENTATION_LINK}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                here
-              </a>
-              .
-            </>
-          )
-        }
-      />
-    </div>
-  );
+  const renderAvailableSelector = () => {
+    let sortedAvailableDBs;
+    if (availableDbs) {
+      sortedAvailableDBs = [...(availableDbs?.databases || [])];
```

Comment:
one last thing.. can you move this down to line 568 and remove a few lines of code that way?